platforms/documentation/docs/src/snippets/dependencyManagement/artifactTransforms-simple/tests/artifactTransformSimple.sample.conf

```diff
@@ -0,0 +1,2 @@
+executable: gradle
+args: tasks
```

Comment:
We should run something other than `tasks` to verify this works. Printing the tasks out will not resolve the `runtimeClasspath` configuration and therefore will not run the transform. 

I would recommend adding a custom resolve method to do so (you can keep it out of the tag:: so its not visible in the snippet) (below is untested but should be close)

```
depenencies {
    implementation("com.google.guava:guava:33.2.1-jre")
}

repositories {
    mavenCentral()
}

tasks.register("resolve") {
    val files = configurations.runtimeClasspath.incoming.files
    doLast {
        assert(files.map { it.name } == listOf("guava-33.2.1-jre.transformed.jar"))
    }
}
```